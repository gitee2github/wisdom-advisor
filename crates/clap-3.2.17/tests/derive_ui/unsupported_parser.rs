use clap::Parser;

#[derive(Parser, Clone, Debug)]
struct Opt {
    #[clap(parse(not_a_valid_parser))]
    value: i8,
}

fn main() {
    println!("{:?}", Opt::parse());
}
