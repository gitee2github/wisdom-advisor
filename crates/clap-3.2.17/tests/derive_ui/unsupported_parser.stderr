error: unsupported parser `not_a_valid_parser`
 --> tests/derive_ui/unsupported_parser.rs:5:18
  |
5 |     #[clap(parse(not_a_valid_parser))]
  |                  ^^^^^^^^^^^^^^^^^^
